docs/conf.py

```diff
@@ -73,7 +72,7 @@
 todo_include_todos = False
 
 # Parent directory of all process definitions:
-autoprocess_process_dir = os.path.normpath(docs_dir + "./../resolwe_bio/processes/")
+autoprocess_process_dir = os.path.join(base_dir, 'resolwe_bio/processes/')
```

Comment:
@mstajdohar, wouldn't `os.path.join(base_dir, 'resolwe_bio', 'processes')` be more portable (cross-platform)?